x-pack/plugins/ml/public/application/components/job_spaces_repair/repair_results.tsx

```diff
@@ -0,0 +1,178 @@
+ */
+
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { EuiText, EuiTitle, EuiAccordion, EuiTextColor, EuiHorizontalRule } from '@elastic/eui';
+
+import { RepairSavedObjectResponse } from '../../../../common/types/saved_objects';
+
+export const RepairList: FC<{ repairItems: RepairSavedObjectResponse | null }> = ({
+  repairItems,
+}) => {
+  if (repairItems === null) {
+    return null;
+  }
+
+  return (
+    <>
+      <SavedObjectsCreated repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+
+      <SavedObjectsDeleted repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+
+      <DatafeedsAdded repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+
+      <DatafeedsRemoved repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+    </>
+  );
+};
+
+const SavedObjectsCreated: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.savedObjectsCreated);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsCreated.title"
+              defaultMessage="Missing job saved objects ({count})"
+              values={{ count: items.length }}
+            />
+          </EuiTextColor>
+        </h3>
+      </EuiTitle>
+      <EuiText size="s">
+        <p>
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsCreated.description"
+              defaultMessage="Jobs exist which do not have accompanying saved objects. These will be created in the current space"
+            />
+          </EuiTextColor>
+        </p>
+      </EuiText>
+    </>
+  );
+  return <RepairItem id="savedObjectsCreated" title={title} items={items} />;
+};
+
+const SavedObjectsDeleted: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.savedObjectsDeleted);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsDeleted.title"
+              defaultMessage="Unmatched job saved objects ({count})"
+              values={{ count: items.length }}
+            />
+          </EuiTextColor>
+        </h3>
+      </EuiTitle>
+      <EuiText size="s">
+        <p>
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsDeleted.description"
+              defaultMessage="Saved objects which do not have an accompanying job. These will be left over when deleting anomaly detection jobs."
```

Comment:
Does the second sentence explain why they exist or what will happen to them? If it's what's going to happen to them (as was the case for the missing saved objects), I'd rewrite like this:
```suggestion
              defaultMessage="If there are saved objects that do not have an accompanying job, they will be deleted."
```

Reply:
updated in https://github.com/elastic/kibana/pull/83320/commits/29c6175bd0c1efa92d388f57dfb34a8fcb23e47e